rules/cross-platform/defense_evasion_timestomp_touch.toml

```diff
@@ -24,24 +27,24 @@ type = "eql"
 
 query = '''
 process where event.type == "start" and
- process.name : "touch" and user.id != "0" and
- process.args : ("-r", "-t", "-a*","-m*") and
- not process.args : ("/usr/lib/go-*/bin/go", "/usr/lib/dracut/dracut-functions.sh", "/tmp/KSInstallAction.*/m/.patch/*")
+ (process.name : "touch" and user.id != "0" and process.args : ("-r", "-t", "-a*","-m*")) or 
+ (process.name: "touch" and process.args:("-acmr" or "/bin/*" or "/opt/*")) and
```

Comment:
Invalid EQL query, my last comment was explaining that using `field : ("arg1", "arg2")` is an OR condition, which is not what you need here. I have explained and shown an example for you here here: https://github.com/elastic/detection-rules/pull/1565#discussion_r755959162